Source/WebCore/accessibility/ios/WebAccessibilityObjectWrapperIOS.mm

```diff
@@ -870,7 +870,7 @@ - (BOOL)isSVGGroupElement
 {
     // If an SVG group element has a title, it should be an accessible element on iOS.
     Node* node = self.axBackingObject->node();
-    if (node && node->hasTagName(SVGNames::gTag) && [[self accessibilityLabel] length] > 0)
+    if (node && WebCore::elementName(node) == ElementName::SVG_g && [[self accessibilityLabel] length] > 0)
```

Comment:
Can be shortened to:

`if (WebCore::elementName(node) == ElementName::SVG_g && [[self accessibilityLabel] length] > 0)`